### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by matesio** [davidrlunu/dots-and-dashes#11]
as per instructions i have all files in /home/$USER/.conky , but after running conky manager nothing shows up.

**Comment by gu3st0** [comment]
using debian testing, tried on mate de,i3wm and openbox, and only the network on the top left is showing up, using opsat 1.10 on conky 1.10.8-1

can you make an install or startup script or something if you have time @davidrlunu ? or maybe something with the actual conky version is making conflicts with your setup

**Comment by davidrlunu** [comment]
Hello and sorry for the delay in answering

...Been away for a very long time and i couldn't have the actual time to
maintain that repo. To quick answer / troubleshoot your problem, I will
start by saying that window manager or DE have nothing to do with conky
itself, except for the root window managment... most of problems appear on
Gnome, becaose reasons.
Your description of the problem is very ambiguous as in "the rest of conky
doesn't appear at all"  because it throws errors, or it technically
occupiues space on the screen but nothing is showing up ? Please do check
if you have an active composite manager and if so, open some of the conky
scripts in the terminal to see if they throw errors.

My best guess is that either your composite manger doesn't work properly,
or you are missing fonts. For the rest, the scripts should work even 5
years from now on!

Let me know if you have any other questions!

D.

**Comment by jetpotion** [comment]
Yeah I too i  moved my conky_opsat but nothing is executing. Does the original files all have to be copied individually? Or just the folder itself
